language/tools/move-cli/tests/sandbox_tests/cyclic_multi_module_publish/args.exp

```diff
@@ -6,3 +6,4 @@ Wrote 175 bytes of module ID's and code
 Command `-p p2 sandbox publish --bundle --override-ordering A --override-ordering C -v`:
 Found 3 modules
 Invalid multi-module publishing: VMError with status INVALID_FRIEND_DECL_WITH_MODULES_IN_DEPENDENCIES at location Module ModuleId { address: 00000000000000000000000000000003, name: Identifier("C") } and message At least one module, 00000000000000000000000000000003::A, appears in both the dependency set and the friend set
+Publishing failed with unexpected error INVALID_FRIEND_DECL_WITH_MODULES_IN_DEPENDENCIES
```

Comment:
Would you mind leaving a TODO item to explain this?

Reply:
This was generated by the explain_publish_error() function.
I did not remove the line 
```
println!("Invalid multi-module publishing: {}", err); 
```
in order to have the other test cases run smoothly